### PR TITLE
Using relative time for last update field. Close: #41

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -52,6 +52,32 @@ export default new Vuex.Store({
   },
   getters: {
     isWeatherGot: state => !!state.currentWeather.description,
+    getRelativeTimeLastUpdate: (state) => {
```
Comment (probil) [inline src/store/index.js]: Getter is not needed for that. Consider using filter

File: src/components/CurrentWeather.vue [hunk 2]
```diff
@@ -65,15 +65,15 @@
       >accuweather.com</a>
     </div>
     <div class="current-weather__row">
-      <p class="current-weather__datetime">Останнє оновлення: {{props.weather.dateTime}}</p>
+      <p class="current-weather__datetime">Останнє оновлення: {{props.lastUpdateTime}}</p>
```
Comment (probil) [inline src/components/CurrentWeather.vue]:
```suggestion
      <p class="current-weather__datetime">Останнє оновлення: {{props.weather.lastUpdateTimeStamp | distanceInWordsToNow }}</p>
```

Check vue filters

File: src/store/index.js [hunk 3]
```diff
@@ -52,6 +52,32 @@ export default new Vuex.Store({
...
+      if (!state.currentWeather.lastUpdateTimeStamp) return 'Невідомо';
+      const currTimeStamp = Date.now();
+      // eslint-disable-next-line max-len
+      const lastUpdateCurrentTimeDelayInSeconds = (state.currentWeather.lastUpdateTimeStamp - currTimeStamp) / 1000;
```
Comment (probil) [inline src/store/index.js]: Consider using `date-fns/formatDistanceToNow`. 
https://date-fns.org/v2.0.1/docs/formatDistanceToNow

**Comment by liubich** [comment]
I will do this in new PR

**Pull request closed without merging**